java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -50,10 +50,11 @@ public interface EventHandler {
      *
      * @note the callback is invoked with this `ColumnVector`'s lock held.
      *
+     * @param cv - a reference to the ColumnVector we are closing
      * @param refCount - the updated ref count for this ColumnVector at the time
      *                 of invocation
      */
-    void onClosed(int refCount);
+    void onClosed(ColumnVector cv, int refCount);
```

Comment:
I assume that an `EventHandler` gets access directly to the current `ColumnVector` object.

Reply:
Not in this case. When this is invoked in in the java class that has implemented `EventHandler`, `this` is the reference to this specifc event handler. In order to get access to `ColumnVector` we would have to subclass `ColumnVector` and then override a protected (and default implemented) `onClosed`. We may gravitate towards something like that to make spill-ability really transparent but I don't think we have thought through that yet.

Reply:
But by passing `cv` here, we do get access to the public interface of `ColumnVector` within the callback.